Tests/Security/Http/Firewall/TwoFactorListenerTest.php

```diff
@@ -399,6 +397,9 @@ public function handle_isAuthFormPath_doNothing(): void
         $this->stubCurrentPath(self::FORM_PATH);
 
         $this->assertNoResponseSet();
+        $this->assertEventsDispatched([
```

Comment:
Github doesn't let me comment on the correct line. Let's please rename the test to `handle_isAuthFormPath_dispatchRequireEvent` since that's what we test now.